README.md

```diff
@@ -51,16 +49,13 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://www.loom.com/share/7f24ada1fc6f40868fc4c9a6e934fb7b)
+- [Live Demo Link](https://www.loom.com/share/c6adcc8020ae40c9a7d64e3bc3f9a937)
```

Comment:
- [ OPTIONAL ]  will suggest that since this link is to your loom video, you change the name to `Loom Video Link` or something else that will let visitors know where the link leads to :wink:  
- That way you will only have one link to the `Live Demo` and one to the `Loom Video`
- eg. `[Loom Video Link](https://www.loom.com/share/c6adcc8020ae40c9a7d64e3bc3f9a937)`